generators/client/templates/vue/src/main/webapp/app/shared/data/data-utils.service.ts.ejs

```diff
@@ -44,25 +44,20 @@ export default class JhiDataUtils extends Vue {
    * Method to open file
    */
   openFile(contentType, data) {
-    if (window.navigator && window.navigator.msSaveOrOpenBlob) {
-      // To support IE and Edge
-      const byteCharacters = atob(data);
-      const byteNumbers = new Array(byteCharacters.length);
-      for (let i = 0; i < byteCharacters.length; i++) {
-        byteNumbers[i] = byteCharacters.charCodeAt(i);
-      }
-      const byteArray = new Uint8Array(byteNumbers);
-      const blob = new Blob([byteArray], {
-        type: contentType
-      });
-      window.navigator.msSaveOrOpenBlob(blob);
-    } else {
-      // Other browsers
-      const fileURL = `data:${contentType};base64,${data}`;
-      const win = window.open();
-      win.document.write(
-        '<iframe src="' + fileURL + '" frameborder="0" style="border:0; top:0px; left:0px; bottom:0px; right:0px; width:100%; height:100%;" allowfullscreen></iframe>');
+    const byteCharacters = atob(data);
+    const byteNumbers = new Array(byteCharacters.length);
+    for (let i = 0; i < byteCharacters.length; i++) {
+      byteNumbers[i] = byteCharacters.charCodeAt(i);
     }
+    const byteArray = new Uint8Array(byteNumbers);
+    const blob = new Blob([byteArray], {
+      type: contentType
+    });
+    const objectURL = URL.createObjectURL(blob);
+    const win = window.open();
+    win.document.write(
+      '<img src=\'' + objectURL + '\'>'
```

Comment:
It only will works with images... 
why not?

```
const win = window.open(objectURL);
win!.onload = function() {
        URL.revokeObjectURL(objectURL);
}  
```

Reply:
oops you're right 👍